### PR TITLE
add FindPositionCommandTests

### PR DESCRIPTION
**Pull request by actimellka**

### CODE REVIEW

File: src/PositionService.Data/PositionRepository.cs [hunk 1]
```diff
@@ -49,9 +49,16 @@ public async Task<(List<DbPosition>, int totalCount)> FindAsync(FindPositionsFil
...
+      if (filter.IsActive.HasValue)
       {
-        dbPositions = dbPositions.Where(p => p.IsActive);
+        dbPositions = dbPositions.Where(x => x.IsActive == filter.IsActive);
+      }
+
+      if (filter.IsAscendingSort.HasValue)
+      {
+        dbPositions = filter.IsAscendingSort.Value
+        ? dbPositions.OrderBy(o => o.Name)
```
Comment (MariyaSin) [inline src/PositionService.Data/PositionRepository.cs:60]: add tab

File: src/PositionService.Models.Dto/Requests/Position/Filters/FindPositionsFilter.cs [hunk 2]
```diff
@@ -1,9 +1,14 @@
...
 namespace LT.DigitalOffice.PositionService.Models.Dto.Requests.Position.Filters
 {
   public record FindPositionsFilter : BaseFindFilter
   {
-    public bool IncludeDeactivated { get; init; } = false;
+    [FromQuery(Name = "isAscendingSort")]
+    public bool? IsAscendingSort { get; set; }
+
+    [FromQuery(Name = "isActive")]
+    public bool? IsActive { get; set; }
```
Comment (MariyaSin) [inline src/PositionService.Models.Dto/Requests/Position/Filters/FindPositionsFilter.cs:12]: please, add
[FromQuery(Name = "nameIncludeSubstring")]
public string NameIncludeSubstring { get; set; }

File: test/PositionService.Business.UnitTests/FindPositionCommandTests.cs [hunk 3]
```diff
@@ -0,0 +1,265 @@
...
+          Id = _ids[2],
+          Name = "TestName3",
+          Description = "TestDescription3",
+          IsActive = true,
+          CreatedBy = Guid.NewGuid(),
+          CreatedAtUtc = DateTime.UtcNow,
+        },
+      };
+
+      _totalCount = 3;
```
Comment (MariyaSin) [inline test/PositionService.Business.UnitTests/FindPositionCommandTests.cs:74]: dbPositions.Count

File: test/PositionService.Business.UnitTests/FindPositionCommandTests.cs [hunk 4]
```diff
@@ -0,0 +1,265 @@
...
+      _mocker.GetMock<IPositionRepository>().Reset();
+      _mocker.GetMock<IPositionInfoMapper>().Reset();
+      _mocker.GetMock<IResponseCreator>().Reset();
+      _mocker.GetMock<IBaseFindFilterValidator>().Reset();
+    }
+
+    [Test]
+    public async Task ShouldReturnListPositionInfoAsync()
+    {
+      var result = _positionInfo;
```
Comment (MariyaSin) [inline test/PositionService.Business.UnitTests/FindPositionCommandTests.cs:102]: do you need the variable?

File: test/PositionService.Business.UnitTests/FindPositionCommandTests.cs [hunk 5]
```diff
@@ -0,0 +1,265 @@
...
+      SerializerAssert.AreEqual(result, (await _command.ExecuteAsync(_filter)).Body);
+
+      _mocker.Verify<IPositionRepository, Task<(List<DbPosition>, int totalCount)>>(x => x.FindAsync(It.IsAny<FindPositionsFilter>()), Times.Once);
+      _mocker.Verify<IPositionInfoMapper, PositionInfo>(x => x.Map(It.IsAny<DbPosition>()), Times.Once);
+    }
+
+    [Test]
+    public async Task ShouldReturnListSortPositionInfoAsync()
+    {
+      var result = _positionInfo;
```
Comment (MariyaSin) [inline test/PositionService.Business.UnitTests/FindPositionCommandTests.cs:172]: same

File: TestProject1/PositionRepositoryTests.cs [hunk 6]
```diff
@@ -0,0 +1,559 @@
...
+      };
+
+      CreateMemoryDb();
+
+      _provider.Positions.AddRange(_position1);
+      _provider.Positions.AddRange(_position2);
+      _provider.Positions.AddRange(_deactivatedPosition);
+      _provider.Positions.AddRange(_positionWithUser);
+      _provider.PositionsUsers.AddRange(_user);
+      _provider.Save();
```
Comment (SolovyovDmitry) [inline TestProject1/PositionRepositoryTests.cs:110]: same

**Review by VLiamin — APPROVED** [review]

**Review by MariyaSin — APPROVED** [review]

**Pull request merged**